packages/components/src/core/InputDropdown/style.ts

```diff
@@ -20,9 +21,11 @@ export interface InputDropdownProps extends CommonThemeProps {
   open?: boolean;
   sdsStage: "default" | "userInput";
   sdsStyle?: "minimal" | "square" | "rounded";
-  sdsType?: "singleSelect" | "multiSelect";
+  sdsType?: "label" | "value";
```

Comment:
looks like somewhere in this file we need to filter out `sdsType` from passing to the DOM 😄 
<img width="1292" alt="Screenshot 2023-07-19 at 12 55 58 PM" src="https://github.com/chanzuckerberg/sci-components/assets/6309723/e3d5f475-3e2e-4c0c-b067-756865ba9d92">

Reply:
Ahh, I just fixed it :D

Reply:
hurray you the best!!